### PR TITLE
[FEATURE] CRUD for teacher_schedule #82

### PR DESCRIPTION
**Pull request by ershadteymurluu**
Added CRUD for teachers_schedule table

### CODE REVIEW

File: src/main/java/com/webperside/courseerpbackend/repository/TeachersScheduleRepository.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import org.apache.ibatis.annotations.Mapper;
+
+import java.util.Collection;
+import java.util.Optional;
+
+@Mapper
+public interface TeachersScheduleRepository {
+    void insert(TeachersSchedule teachersSchedule);
+    Optional<TeachersSchedule> findById(Long id);
+    Collection<TeachersSchedule> findAll();
```
Comment (hamidsultanzadeh) [inline src/main/java/com/webperside/courseerpbackend/repository/TeachersScheduleRepository.java:13]: Use List here

File: src/main/java/com/webperside/courseerpbackend/models/mybatis/teacher/TeachersSchedule.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+@EqualsAndHashCode(callSuper = true)
+@Data
+@SuperBuilder
+@NoArgsConstructor
+@AllArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class TeachersSchedule extends BaseEntity<Long> {
+    Long branchId;
+    String dayOfWeek;
```
Comment (hamidsultanzadeh) [inline src/main/java/com/webperside/courseerpbackend/models/mybatis/teacher/TeachersSchedule.java:19]: Use DayOfWeek enum here

File: src/main/java/com/webperside/courseerpbackend/services/teacher/TeachersScheduleService.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+import com.webperside.courseerpbackend.models.mybatis.teacher.TeachersSchedule;
+
+import java.util.Collection;
+import java.util.Optional;
+
+public interface TeachersScheduleService {
+    void insert(TeachersSchedule teachersSchedule);
+    TeachersSchedule findById(Long id);
+    Collection<TeachersSchedule> findAll();
```
Comment (hamidsultanzadeh) [inline src/main/java/com/webperside/courseerpbackend/services/teacher/TeachersScheduleService.java:11]: Use List here

**Review by hamidsultanzadeh — CHANGES_REQUESTED** [review]

File: src/main/java/com/webperside/courseerpbackend/repository/TeachersScheduleRepository.java (hunk 1 above)
Comment (ershadteymurluu) [reply src/main/java/com/webperside/courseerpbackend/repository/TeachersScheduleRepository.java]: Changes made, pls consider it.

File: src/main/java/com/webperside/courseerpbackend/models/mybatis/teacher/TeachersSchedule.java (hunk 2 above)
Comment (ershadteymurluu) [reply src/main/java/com/webperside/courseerpbackend/models/mybatis/teacher/TeachersSchedule.java]: Changes added according to your review. Please consider it again, thanks

**Review by hamidsultanzadeh — APPROVED** [review]

**Review by aytiqaqash — APPROVED** [review]
LGTM

**Pull request merged**